src/styles/base/index.scss

```diff
@@ -14,6 +14,30 @@ a:hover {
   padding: 34px 0 34px 0;
 }
 
+h1 {
+  font-size: 60px;
+  font-weight: 800;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.2;
+  letter-spacing: 6px;
+  text-align: left;
+  color: #000000;
+  text-align: left;
+}
+
+p  {
+  font-size: 16.5px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.73;
+  letter-spacing: normal;
+  text-align: left;
+  margin: 36px 187px 58px 2px;
```

Comment:
don't use margin for your base paragraph styles, it will affect all your `p` not only in your section